build.ps1

```diff
@@ -284,10 +305,16 @@ try {
         if (IsWindowsDevelopmentBox)
         {
             RunTests e2e\test "End-to-end tests (NetCoreApp2.1, NET47, NET451)"
+
+            # To exclude the Pooling Fault Injection Tests from E2E test run (Pooling Fault Injection E2E tests require an additional IoTHub):
+            # RunTests e2e\test "End-to-end tests (NetCoreApp2.1, NET47, NET451)" "*" "TestCategory!=IoTHub-FaultInjection-PoolAmqp"
```

Comment:
Why are these commented out?

Reply:
We are back to running all non-pooled-fault injection tests and then pooled-fault-injection tests.